test/cql/cdc_too_short_stream_id_test.cql

```diff
@@ -0,0 +1,6 @@
+create table tb (pk int primary key) with cdc = {'enabled': true};
+
+-- Key of length != 128 b should return empty result set (issue #6570)
+select * from ks.tb_scylla_cdc_log where "cdc$stream_id" = 0x00;
+
+select * from ks.tb_scylla_cdc_log where "cdc$stream_id" = 0x;
```

Comment:
`tb_scylla_cdc_log` is empty, so it is not surprising that the result is empty... While I am aware that the fix is against a segfault occuring here, I think that we should probably make the cdc log non-empty so that this test guards against bugs which could cause a non-empty result set to be returned in this case.

Reply:
Good point!

Reply:
done!